Berksfile

```diff
@@ -1,6 +1,7 @@
 source 'https://supermarket.chef.io'
 
 # cookbook 'percona', '>= 0.15.0'
+cookbook 'base', git: 'git@github.com:osuosl-cookbooks/base'
 cookbook 'percona', github: 'phlipper/chef-percona'
```

Comment:
Let's no longer rely on the github repo and remove this and the reference above.

Reply:
@ramereth Do you mean line 3-5?
For line 4, if it is deleted, chef uses base cookbook from the supermarket.

Reply:
I mean the line which contains ``percona``

Reply:
@JerryPeng0112 you still need to remove this line.